packages/labs/analyzer/src/lib/javascript/functions.ts

```diff
@@ -70,20 +91,24 @@ const getFunctionDeclaration = (
  */
 export const getFunctionLikeInfo = (
   node: ts.FunctionLikeDeclaration,
+  docNode: ts.FunctionLikeDeclaration,
   analyzer: AnalyzerInterface
 ) => {
   return {
-    parameters: node.parameters.map((p) => getParameter(p, analyzer)),
-    return: getReturn(node, analyzer),
+    parameters: node.parameters.map((p, i) =>
+      getParameter(p, docNode.parameters[i], analyzer)
```

Comment:
This is why I need to check if `.parameters.length` can vary between override declarations.

Reply:
This was removed.